packages/internals/console/src/utils.test.js

```diff
@@ -0,0 +1,66 @@
+const {expect} = require('../../../../test/helpers');
+const utils = require('./utils');
+const parse = require('./parse');
+
+describe('console utils', function () {
+  describe('.countLeadingEmptyLines()', function () {
+    it('counts the number of blank lines at the start of a string', function () {
+      expect(utils.countLeadingEmptyLines(`\n\n\nfoo\n\nbar\n`)).to.equal(3);
+      expect(utils.countLeadingEmptyLines(`foo\n\nbar\n`)).to.equal(0);
+      expect(utils.countLeadingEmptyLines(`\n\n`)).to.equal(2);
+    });
+  });
+
+  describe('.parseError()', function () {
+    it('returns an object with message and stack properties', function () {
+      const result = utils.parseError(new Error('foobar'));
+      expect(result).to.have.property('message');
+      expect(result).to.have.property('stack');
+      expect(result.message).to.be.a('string');
+      expect(result.stack).to.be.a('string');
+    });
+    it('has a null stack property if the input is not an Error instance', function () {
+      const result = utils.parseError('foobar');
+      expect(result.message).to.equal('foobar');
+      expect(result.stack).to.equal(null);
+    });
+  });
+
+  describe('.prefix()', function () {
+    it('adds a prefix to the start of a string', function () {
+      expect(utils.prefix('abcdefg', '123 ')).to.equal('123 abcdefg');
+    });
+    it('prefixes the first non-empty line and indents the other lines to match', function () {
+      expect(utils.prefix('\n\nabcde\nfg\nhi', '!')).to.equal('\n\n!abcde\n fg\n hi');
```

Comment:
With a name like prefix, I would expect the first TBH. I think the problem is just the name, as I can see what it's doing and why it's useful.
It's more akin to a hanging indent...